packages/svgo-jsx/svgo-jsx.js

```diff
@@ -96,7 +96,7 @@ const convertAttributes = (node, svgProps) => {
       props.set(name, JSON.stringify(value));
     }
   }
-  if (node.name === "svg" && svgProps) {
+  if ((node.name === "svg" || node.name === "Svg") && svgProps) {
```

Comment:
toLowerCase() to cover all possible cases at ones

Reply:
Maybe will be better to check the hierarchy not an element name in this case? The SVG component can be named in various ways (Root, SVGRroot, whatever), so we inject the props only for the top-level element.

Reply:
Yep, I guess it's find to spread into an element in the root. Then need to pass parentNode and check it's type for "root".

Reply:
Done, I'm not sure about argument order or maybe you have other suggestions, welcome 🙌🏻